pkgs/by-name/holo-cli/default.nix

```
@@ -0,0 +1,19 @@
+{
+  pkgs ? import <nixpkgs> {
```

This [nixpkgs lookup path](https://nix.dev/guides/best-practices.html#lookup-paths) might be different on two machines, which is not reproducible. We already have `pkgs` available, which uses nixpkgs from NGIpkgs' flake and we can just use that, instead.

```suggestion
  pkgs,
```